oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -689,6 +820,78 @@ void pldm::responder::oem_ibm_platform::Handler::setSurvTimer(uint8_t tid,
     }
 }
 
+void pldm::responder::oem_ibm_platform::Handler::
+    processPowerCycleOffSoftGraceful()
+{
+    error("Received soft graceful power cycle request");
+    pldm::utils::PropertyValue value =
+        "xyz.openbmc_project.State.Host.Transition.ForceWarmReboot";
+    pldm::utils::DBusMapping dbusMapping{"/xyz/openbmc_project/state/host0",
+                                         "xyz.openbmc_project.State.Host",
+                                         "RequestedHostTransition", "string"};
+    try
+    {
+        dBusIntf->setDbusProperty(dbusMapping, value);
+    }
+    catch (const std::exception& e)
+    {
+        error(
+            "Error to do a ForceWarmReboot, chassis power remains on, and boot the host back up. Unable to set property RequestedHostTransition. ERROR={ERR_EXCEP}",
```

Comment:
just `e` is enough no need of `e.what()`

Reply:
done